controllers/account-setting/my-team/change-access-request-status.controller.js

```diff
@@ -5,7 +5,16 @@ const { addNotificationFunc } = require('../../../services/notification.service'
 
 const changeAccessRequestStatus = async (req, res, next) => {
 	const user = req.user;
+
 	try {
+		// Check if the user has the required role
+		if (user.role !== 'CRM_ADMIN') {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth